packages/core/src/presentation/forms/validation/validators.ts

```diff
@@ -85,6 +85,12 @@ const skipIfUndefined = (actualValidator: IValidator): IValidator => {
   };
 };
 
+const skipIfSpel = (actualValidator: IValidator): IValidator => {
+  return function skipIfSpel(val: any, label = THIS_FIELD) {
+    return typeof val === 'string' && val.indexOf('${') !== 0 && actualValidator(val, label);
```

Comment:
Sorry if I'm being dense, what's the reason for checking the value doesn't start with ${?